### PR TITLE
Added PadlePadle bitwise_right_shift function 

### PR DESCRIPTION
**Pull request by vandit98**
Added bitwise_right_shift to Paddle Backend.

### CODE REVIEW

**Comment by vandit98** [comment]
Hi @MahmoudAshraf97 
With reference to above pull request i have tried to implement bitwise_right_shift in  paddle backend.
https://github.com/unifyai/ivy/pull/10935/files
Please let me know if i am doing it right.
Thanks

**Comment by MahmoudAshraf97** [comment]
Hi @vandit98 , unfortunately the implementation is not correct because the function you used doesn't exist in paddle, please check the documentation [Here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/index_en.html) and look for an appropriate function.
I didn't find any so probably you'll have to implement it yourself using composite functions

File: ivy/functional/backends/paddle/elementwise.py [hunk 1]
```diff
@@ -467,3 +467,21 @@ def isreal(
...
+                        *, 
+                        out: Optional[paddle.Tensor] = None) -> paddle.Tensor:
+    # Convert input to paddle.Tensor if it is not already
+    if not isinstance(x1, paddle.Tensor):
+        x1 = paddle.to_tensor(x1)
+    if not isinstance(x2, paddle.Tensor):
+        x2 = paddle.to_tensor(x2)
+    
+    # Perform bitwise right shift operation
+    result = paddle.fluid.layers.bitwise_right_shift(x1, x2, out)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/elementwise.py:484]: this function doesn't exist

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]

**Comment by vandit98** [comment]
hi @MahmoudAshraf97 
I have tried another method.
Can You please review it.
Thanks

**Review by MahmoudAshraf97 — APPROVED** [review]

**Comment by MahmoudAshraf97** [comment]
Great work, i've added some casting to ensure compatibility in some operations so please make sure they're added the next time, will proceed to merge this

**Pull request merged**